source/installer.py

```diff
@@ -633,7 +662,7 @@ def install(shouldCreateDesktopShortcut: bool = True, shouldRunAtLogon: bool = T
 	# so we shouldn't proceed.
 	# 2. The appropriate executable for nvda.exe will be determined by
 	# which executables exist after copying program files.
-	for f in ("nvda.exe","nvda_noUIAccess.exe","nvda_UIAccess.exe","nvda_service.exe","nvda_slave.exe"):

```

Comment:
There is a change of logic here - we no longer check for `nvda_service.exe`. While it is true NVDA no longer installs the service for some time now, it is possible, though unlikely, someone upgrades from the version where service was included. In general given this targets a patch release I would be pretty careful with introducing any logic changes, unless they are strictly required to fix the issue this PR intents to solve.